### PR TITLE
[TechDocs] Enhance PlantUML support in the techdocs container to include external puml or pu files

### PR DESCRIPTION
**Pull request by lavanya-sainik-ericsson**
Issue Reference #20184

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [X] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [X] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: docs/features/techdocs/how-to-guides.md [hunk 1]
```diff
@@ -525,6 +525,39 @@ techdocs:
 This way, all iframes where the host in the src attribute is in the
 `sanitizer.allowedIframeHosts` list will be displayed.
 
+## How to render plantUML diagram in TechDocs
```
Comment (awanlin) [inline docs/features/techdocs/how-to-guides.md:528]:
```suggestion
## How to render PlantUML diagram in TechDocs
```

File: docs/features/techdocs/how-to-guides.md [hunk 2]
```diff
@@ -525,6 +525,39 @@ techdocs:
...
+
+PlantUML allows to create diagrams from plain text language. Each diagram description begins with the keyword - (@startXYZ and @endXYZ, depending on the kind of diagram). For UML Diagrams, Keywords @startuml & @enduml should be used. Further details for all types of diagrams can be found at [PlantUML Language Reference Guide](https://plantuml.com/guide).
```
Comment (awanlin) [inline docs/features/techdocs/how-to-guides.md:530]:
```suggestion
PlantUML allows you to create diagrams from plain text language. Each diagram description begins with the keyword - (@startXYZ and @endXYZ, depending on the kind of diagram). For UML Diagrams, Keywords @startuml & @enduml should be used. Further details for all types of diagrams can be found at [PlantUML Language Reference Guide](https://plantuml.com/guide).
```

Comment (awanlin) [inline docs/features/techdocs/how-to-guides.md:530]:
```suggestion
PlantUML allows you to create diagrams from plain text language. Each diagram description begins with the keyword - (@startXYZ and @endXYZ, depending on the kind of diagram). For UML Diagrams, Keywords @startuml & @enduml should be used. Further details for all types of diagrams can be found at [PlantUML Language Reference Guide](https://plantuml.com/guide).
```

**Review by awanlin — CHANGES_REQUESTED** [review]
Thanks for this contribution @lavanya-sainik-ericsson, just left a few small corrections 👍

**Comment by lavanya-sainik-ericsson** [comment]
Hello [awanlin](https://github.com/awanlin),

Thanks for your comments. Have done those changes. Let me know if any more corrections needed or it can be merged.

**Review by johnphilip283 — APPROVED** [review]

**Review by awanlin — APPROVED** [review]

**Comment by awanlin** [comment]
@johnphilip283 thanks for the review and approval. Feel free to merge when you have time, that's not something I've generally done

**Pull request merged**